### ISSUE TITLE
Add Hound to the repo

### ISSUE THREAD

**Issue by maebeale** [rubyforgood/voices-of-consent#342]

**Comment by PacoLuna1** [comment]
@maebeale  can I try it?

**Comment by maebeale** [comment]
Sure can! Thanks, @PacoLuna1 !

**Comment by PacoLuna1** [comment]
Can you be little more specific about what you want? @maebeale

**Comment by maebeale** [comment]
@PacoLuna1 Someone else has a PR adding rubocop to the repo, and we want to add Hound so that it'll run rubocop on any PRs. Any other settings or Hound implementation you recommend?

**Comment by atzorvas** [comment]
@maebeale shouldn't hound be installed using this link from a repo owner?  https://github.com/marketplace/hound

**Comment by PacoLuna1** [comment]
yes thats why I have doubts, I only can install hound to my fork
![image](https://user-images.githubusercontent.com/37641348/66776518-2ed42d00-ee8c-11e9-888e-1557de9ce91d.png)

**Comment by PacoLuna1** [comment]
@maebeale it is a great idea to install scss-lint to keep your SCSS files clean and readable
https://github.com/sds/scss-lint
